Update no-unused-prop-types.md (#2273)
yannickcr/eslint-plugin-react: docs/rules/no-unused-prop-types.md

  bProp: PropTypes.string // bProp is defined but never used
};

> ```

Reviewer: i prefer to keep the term SFC while it's still in common usage in the community. Please revert these.

Author: There's no mention of SFC in the react docs anymore. Or stateless function. https://www.dropbox.com/s/u51xabyw7wfuqqy/Screenshot%202019-05-12%2008.12.19.png?dl=0. And even if one does know the history and what SFC means, the warnings apply to function components with state (via Hooks) too.

Reviewer: React docs are irrelevant; it’s still a widely used term in the community. A component with a useState hook is still an SFC.

};
```

Using intermediate variables might be desired and unavoidable for more complex props structure.

> Like for shape propTypes. To avoid false positive in this case make sure `skipShapeProps` is set to `true`.

Reviewer: ```suggestion
For example, for shape propTypes, to avoid a false positive, make sure `skipShapeProps` is set to `true`.
```
